### DISCUSSION TITLE
Enhancing comparability with unequal number of categories?

### DISCUSSION THREAD

**Discussion by ToreWentzel-Larsen** [amices/mice#373] [Ideas]
Working through section 9.4 on Enhancing comparability, I have thought about whether the procedure could be used if the number of categories in the two questions is not the same. In the main example in section 9.4 the two questions both have 4 categories, but the procedure described seems to be possible to use without this restriction. If possible this could be of interest for revisions of instruments, such as from DSM-4 to DSM-5, or from ICD10 to ICD11.

**Comment by stefvanbuuren** [comment]
Yes, that's perfectly possible. If you're interested in this type of harmonisation, you may wish to look at <https://github.com/stefvanbuuren/harmonisation_warsaw_2019/blob/master/Lecture_2/Lecture_2.pdf>.
